components/brave_wallet_ui/components/buy-send-swap/create-account/index.tsx

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+// Redux
+import { useSelector, useDispatch } from 'react-redux'
+import { WalletActions } from '../../../common/actions'
+import { PanelActions } from '../../../panel/actions'
+
+// Types
+import { BraveWallet, WalletState } from '../../../constants/types'
+
+// Components
+import { NavButton } from '../../extension'
+
+// Utils
+import { getLocale } from '../../../../common/locale'
+
+// Styled Components
+import {
+  StyledWrapper,
+  Description,
+  ButtonRow
+} from './style'
+
+export interface Props {
+  isPanel?: boolean
+  prevNetwork: BraveWallet.NetworkInfo | undefined
```

Comment:
Can we use the `previous-network` hook in the component instead of passing it as a prop?

Reply:
`previous-network` needs to be called before this component renders, else it wont store the previous network.

Reply:
👍